app/components/DirectMessageForm.tsx

```diff
@@ -76,9 +76,15 @@ export function DirectMessageForm({
 
     // send message
     if (legacy) {
-      dms.send(replyTo, content)
+      const ev = await dms.send(replyTo, content)
+      if (!ev.id) {
+        alert("Failed to publish")
```

Comment:
this is fine, but we should do this:

https://github.com/ArcadeLabsInc/arcade/issues/232

the queue can use the mobx/storage or can use the db (faster)